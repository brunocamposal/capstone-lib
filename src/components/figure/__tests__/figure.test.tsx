import React from 'react'
import Figure from '../index'
import renderer from 'react-test-renderer'

describe('Figure', () => {
  it('should render a Figure', () => {
    const tree = renderer
    .create(
      <Figure
        src='https://440245-1378968-raikfcquaxqncofqfm.stackpathdns.com/wp-content/uploads/2019/01/fatos-curiosos-sobre-o-homer-simpson.jpg'
        alt='Homer'
        definition='Simpson'
        x={120}
      />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
