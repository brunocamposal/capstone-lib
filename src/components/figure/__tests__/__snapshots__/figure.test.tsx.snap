// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Figure should render a Figure 1`] = `
<figure
  className="sc-bdnylx bGApif"
>
  <img
    alt="Homer"
    className="sc-gtssRu bRcjfP"
    src="https://440245-1378968-raikfcquaxqncofqfm.stackpathdns.com/wp-content/uploads/2019/01/fatos-curiosos-sobre-o-homer-simpson.jpg"
    width={120}
  />
  <figcaption
    className="sc-dlnjPT llcwuW"
  >
    “
    Homer
    ”
  </figcaption>
</figure>
`;
